x-element.js

```diff
@@ -735,16 +735,19 @@ export default class XElement extends HTMLElement {
     // Process possible sources of initial state, with this priority:
     // 1. imperative, e.g. `element.prop = 'value';`
     // 2. declarative, e.g. `<element prop="value"></element>`
-    const { key, attribute } = property;
+    const { key, attribute, internal } = property;
     let value;
     let found = false;
```

Comment:
Note that we will now _always_ return `found = false` when looking up internal properties on the pre-upgraded element’s public interface — because, obviously, they’re not on the public interface…